docs/modelserving/v1beta1/amd/README.md

```diff
@@ -0,0 +1,189 @@
+# AMD Inference Server
```

Comment:
This isn't exposed on website unless user dig into the this github repo, I am thinking that we can use this as an example for how to add a new serving runtime in addition to the default installed ones.

Reply:
Oh I see... I need to modify mkdocs.yml as well to make it visible. I think it should go in the multi-framework serving runtimes. If you'd like other documentation for how to add a new runtime somewhere, I can help write that too

Reply:
I've added it to mkdocs.yml to show it on the website